bulldozer/merge.go

```diff
@@ -195,3 +197,30 @@ func allCommits(ctx context.Context, pullCtx pull.Context, client *github.Client
 
 	return repositoryCommits, nil
 }
+
+func postCommentIfUserActionable(ctx context.Context, client *github.Client, owner, repo string, prNum int, githubError github.ErrorResponse) {
+	commentBody := "Error occurred when attempting merge: " + githubError.Message
```

Comment:
I would be nice to make this more conversational and nicely formatted, maybe:

```
I wasn't able to merge this pull request! GitHub returned this error:

> {{githubError.Message}}

This can happen when {{reasonThisCanHappen}}.
```